debs/focal/fits/Makefile

```diff
@@ -0,0 +1,51 @@
+NAME          = fits
+VERSION       ?= 1.1.0
+RELEASE	      ?= 6~20.04
+GIT_URL	      = https://github.com/harvard-lts/fits
+GPG_ID        ?= 5236CA08
+DEB_TOPDIR    = "/debbuild"
+DOCKER_VOLUME = "/src"
+DOCKER_IMAGE  = "debbuild-$(NAME)-$(VERSION)"
+
+.PHONY: build-docker-image build deb-build deb-clean deb-test update-changelog
+
+all: build-docker-image build
+
+build-docker-image:
+	@echo "==> Building Docker image with build environment."
+	@docker build --tag "$(DOCKER_IMAGE)" --build-arg GIT_URL="$(GIT_URL)" --build-arg GIT_BRANCH="$(VERSION)" --build-arg PACKAGE="$(NAME)" .
+
+update-changelog:
+	@echo "==> Update changelog."
+	@dch -v $(VERSION)-$(RELEASE) -D xenial "New upstream release"
+
+build:
+	@echo "==> Building deb."
+	@docker run --rm \
+		-e GPG_ID=$(GPG_ID) \
+		-e GPG_KEY \
+ 		--volume "$(shell pwd):$(DOCKER_VOLUME)" $(DOCKER_IMAGE) make -C $(DOCKER_VOLUME) deb-build
+
+deb-build: deb-clean
+	@echo "===> Install gpg key"
+	if [ -f "$(DOCKER_VOLUME)/GPG-KEY" ]; then gpg --import $(DOCKER_VOLUME)/GPG-KEY; fi
+	if [ x"$$GPG_KEY" != x ]; then echo "$$GPG_KEY" | gpg --import - ; fi
+	@echo "==> Apply patches."
+	cd /debbuild/$(NAME) && QUILT_PATCHES=debian/patches/ quilt push -a 
+	@echo "==> Install dependencies."
+	cd /debbuild/$(NAME) && yes | mk-build-deps --install debian/control
+	@echo "==> Update changelog."
+	cd /debbuild/$(NAME) && dch -v $(VERSION)-$(RELEASE) -D xenial "New upstream release"
+	@echo "==> Build package."
+	cd /debbuild/$(NAME) && dpkg-buildpackage -k$(GPG_ID)
```

Comment:
When I try to build the `fits` packages (either in `focal` or `jammy`) I keep getting signing issues:

```
==> Build package.
cd /debbuild/fits && dpkg-buildpackage -k5236CA08
dpkg-buildpackage: info: source package fits
dpkg-buildpackage: info: source version 1.1.0-6~22.04
dpkg-buildpackage: info: source distribution jammy
dpkg-buildpackage: info: source changed by root <root@966c24a84b97>
dpkg-buildpackage: error: short OpenPGP key IDs are broken; please use key fingerprints in -k or DEB_SIGN_KEYID instead
dpkg-buildpackage: info: host architecture amd64
```

And the `bulk_extractor` packages that don't have the `-k$(GPG_ID)` option build fine. Maybe it's not important when this gets build in Jenkins?

Reply:
I usually create new packages on my computer with:
```
GPG_ID=XXXXXXXXXXXXXX make
```

I just added the GPG_ID to the `bulk_extractor` package (bionic, focal and jammy)